From 839bf28ee584cce85cafac543d7eb4037fec2780 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Mon, 7 Feb 2022 17:08:56 +0000
Subject: [PATCH] Do not build the man pages

---
 docs/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/Makefile b/docs/Makefile
index c9bf585..896fe63 100644
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -38,7 +38,7 @@ MAN3TARGETS = efi_append_variable.3 \
 	     efi_variable_set_attributes.3 \
 	     efi_variable_realize.3
 
-all : $(MAN1TARGETS) $(MAN3TARGETS)
+all :
 
 clean :
 	@rm -f efisecdb.1
@@ -47,7 +47,7 @@ prep :
 
 test :
 
-install : $(MAN1TARGETS) $(MAN3TARGETS)
+install :
 	$(INSTALL) -d -m 755 $(DESTDIR)$(MANDIR)/man1
 	$(foreach x, $(MAN1TARGETS), $(INSTALL) -m 644 $(x) $(DESTDIR)/$(MANDIR)/man1/;)
 	$(INSTALL) -d -m 755 $(DESTDIR)$(MANDIR)/man3
-- 
2.34.1

